### ISSUE TITLE
Fatal error when live preview Block Themes

### ISSUE THREAD

**Issue by dericleeyy** [woocommerce/woocommerce-blocks#10770]

**Comment by tarhi-saad** [comment]
Hello, @rrennick! 👋 Thanks for forwarding this ticket. 🙌 I've been able to replicate the problem. However, I'm uncertain if it's directly tied to the WooCommerce Blocks. Could you provide some insight into why you believe this issue stems from the WooCommerce Blocks feature?

Cc: @dinhtungdu

**Comment by dinhtungdu** [comment]
@tarhi-saad Yup. Looking at the stack trace, I don't see any reference to WC Blocks as well. I tested with WooCommerce 7.0.0 and can reproduce the same issue.

<img width="850" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/5423135/9a0e3ff9-2dcf-4bac-b070-4e01016e5003">

**Comment by rrennick** [comment]
> However, I'm uncertain if it's directly tied to the WooCommerce Blocks.

@tarhi-saad Thanks for researching this. You can transfer it back to the core repo if it needs to be addressed there.
